pypeit/pypeit.py

```diff
@@ -785,13 +794,17 @@ def objfind_one(self, frames, det, bg_frames, std_outfile=None):
                 slits=self.caliBrate.slits,  # For flexure correction
                 ignore_saturation=False), frame_par['process'])
 
+        # Deal with manual extraction
+        ifile0 = np.where(self.fitstbl['filename'] == os.path.basename(sci_files[0]))[0][0]
```

Comment:
I think the `frames` argument to this function contains the row indices in `fitstbl` for the frames to reduce, not the file names.  So I don't think you need to search and find the row indices, I think `frames` gives them to you directly.